code/modules/uplink/uplink_items_fulp.dm

```diff
@@ -105,6 +105,15 @@
 	desc = "Robust combat boots especially for lizardmen. Perfect for walking over piled human corpses. This pair of shoes is specifically designed to prevent slipping on wet surfaces."
 	clothing_flags = NOSLIP
 
+/datum/uplink_item/role_restricted/clockwork
+	name = "Justiciar's Follower Kit"
+	desc = "Before a select Cult vanished from the system, \
```

Comment:
"cult" shouldn't be capitalized

this first line is also worded pretty awkwardly; consider changing it